src/sidebar/components/test/annotation-test.js

```diff
@@ -1068,22 +1068,57 @@ describe('annotation', function() {
     });
 
     describe('tag display', function () {
-      it('displays links to tags on the stream', function () {
+      beforeEach('make serviceUrl() return a URL for the tag', function() {
         fakeServiceUrl
           .withArgs('search.tag', {tag: 'atag'})
           .returns('https://test.hypothes.is/stream?q=tag:atag');
+      });
 
-        var directive = createDirective(Object.assign(fixtures.defaultAnnotation(), {
+      /*
```

Comment:
If this is meant to be a JSDoc comment as opposed to just an inline comment, it needs to start with `/**` rather than `/*`.

Reply:
Oops, yeah that looks like a typo